api/discovery/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package discovery provides mechanisms for discovery config API servers.
```

Comment:
Doc block is missing something here. Either "for _discovering_ config API servers" or "for discovery _of_ config API servers".
```suggestion
// Package discovery provides mechanisms for discovery of config API servers.
```